mso/resource_mso_site.go

```diff
@@ -168,56 +169,94 @@ func resourceMSOSiteCreate(d *schema.ResourceData, m interface{}) error {
 		siteAttr.Name = name.(string)
 	}
 
-	if username, ok := d.GetOk("username"); ok {
-		siteAttr.ApicUsername = username.(string)
-	}
-
-	if password, ok := d.GetOk("password"); ok {
-		siteAttr.ApicPassword = password.(string)
-	}
-
 	if apic_site_id, ok := d.GetOk("apic_site_id"); ok {
 		siteAttr.ApicSiteId = apic_site_id.(string)
 	}
 
-	if labels, ok := d.GetOk("labels"); ok {
-		siteAttr.Labels = labels.([]interface{})
-	}
-
-	if maintMode, ok := d.GetOk("maintenance_mode"); ok {
-		siteAttr.MaintenanceMode = maintMode.(bool)
-	}
-
-	if domain, ok := d.GetOk("login_domain"); ok {
-		domainStr := domain.(string)
-		usrName := d.Get("username").(string)
-		siteAttr.ApicUsername = fmt.Sprintf("apic#%s\\\\%s", domainStr, usrName)
-		siteAttr.Domain = domainStr
-		siteAttr.HasDomain = true
-	}
-
-	var loc *models.Location
-	if location, ok := d.GetOk("location"); ok {
-		loc = &models.Location{}
-		loc_map := location.(map[string]interface{})
-		loc.Lat, _ = strconv.ParseFloat(fmt.Sprintf("%v", loc_map["lat"]), 64)
-		loc.Long, _ = strconv.ParseFloat(fmt.Sprintf("%v", loc_map["long"]), 64)
-		siteAttr.Location = loc
-	}
-
-	if urls, ok := d.GetOk("urls"); ok {
-		siteAttr.Url = urls.([]interface{})
-	}
+	var apiVersion string
```

Comment:
Why is apiVersion declared 4 times?

Reply:
we declare these variables like msoClient, siteAttr in each resourceMSOSiteCreate/resourceMSOSiteUpdate/resourceMSOSiteRead/resourceMSOSiteDelete func. I suppose they are not duplicate